src/lcell.rs

```diff
@@ -49,13 +49,36 @@ impl<'id> LCellOwner<'id> {
     /// only within the scope of the execution of the given closure
     /// call.  If two scope calls are nested, then the two owners get
     /// different lifetimes.
+    ///
+    /// ```rust
+    /// use qcell::{LCellOwner, LCell};
+    /// LCellOwner::scope(|owner| {
+    ///     let cell = LCell::new(100);
+    ///     assert_eq!(cell.ro(&owner), &100);
+    /// })
+    /// ```
     pub fn scope<F>(f: F)
     where
         F: for<'scope_id> FnOnce(LCellOwner<'scope_id>),
     {
         f(Self { _id: PhantomData })
     }
 
+    /// Create a new `LCellOwner` with a unique lifetime from a `Guard`.
+    /// Requires the `generativity` feature.
+    ///
+    /// ```rust
+    /// use qcell::{generativity::make_guard, LCellOwner, LCell};
+    /// make_guard!(guard);
+    /// let mut owner = LCellOwner::new(guard);
+    /// let cell = LCell::new(100);
+    /// assert_eq!(cell.ro(&owner), &100);
+    /// ```
+    #[cfg(feature = "generativity")]
```

Comment:
If you add `#[cfg_attr(docsrs, doc(cfg(feature = "generativity")))]` to the places in the public API with feature guards, it'll show up nicely on docs.rs (I did this previously with the other feature flags).